src/TreeBuilder.cpp

```diff
@@ -741,7 +731,7 @@ void TreeBuilder::processContainer(const Variable& variable, Node& node) {
         // Contents aren't actually stored inline in this case,
         // but we set this to `true` so that we don't double-count
         // this string's data if we have seen it before.
-        contentsStoredInline = !shouldProcess(*node.pointer);
+        contentsStoredInline = (*node.pointer != 0);
```

Comment:
How does this work? Won't node.pointer always be set?

Reply:
It's completely wrong - my bad! It also fixes that test I skipped once this is done properly.